src/CarbonAware.WebApi/src/Controllers/CarbonAwareController.cs

```diff
@@ -204,6 +204,58 @@ public IActionResult BatchForecastData(IEnumerable<EmissionsForecastBatchDTO> re
         return Ok(result);
     }
 
+    /// <summary>
+    /// Retrieves the actual data that occurred between the time boundaries and calculates the carbon intensity that occurred during that window. 
+    /// </summary>
+    /// <remarks>
+    ///  This endpoint is useful for reporting the measured carbon intensity for a specific time period in a specific location.
+    /// </remarks>
+    /// <param name="location"> the location in </param>
+    /// <param name="jobStartTime"> the time at which the job and thus carbon usage starts </param>
```

Comment:
The average marginal carbon intensity is what it is for that location and time period regardless of anything else.  I understand the desire for clarity based on the ongoing confusions around other `startTime` params, but this feels like it is named too specifically to the single use-case we have in mind and not the overall value of this data.

I think `startTime` / `endTime` would be fine for the context of this endpoint.  Other considerations:
- `time` / `toTime` to match other endpoints
- `periodStartTime` / `periodEndTime` or `averagePeriodStart` / `averagePeriodEnd` to try to describe what the dates represent broadly.

Reply:
since everyone is seems to be okay with start / end time changing it to that